### PR TITLE
Feature/otc 895

### PR DESCRIPTION
**Pull request by lruzicki**
Ticket:
https://openimis.atlassian.net/browse/OTC-895

Changes:
- ceiling type is now fetched from the state when it is possible
- ceiling type is being passed to the backend

Related:
https://github.com/openimis/openimis-be-product_py/pull/27

### CODE REVIEW

File: src/components/ProductForm/DeductiblesCeilingsTabForm.js [hunk 1]
```diff
@@ -452,4 +453,6 @@ const DeductiblesCeilingsTabForm = (props) => {
     </Grid>
   );
 };
+
+
```
Comment (olewandowski1) [inline src/components/ProductForm/DeductiblesCeilingsTabForm.js:457]: Redundant line.

Comment (lruzicki) [reply src/components/ProductForm/DeductiblesCeilingsTabForm.js:457]: fixed

**Review by olewandowski1 — COMMENTED** [review]
I left you one comment, just a cosmetic change. @lruzicki

**Comment by lruzicki** [comment]
> I left you one comment, just a cosmetic change. @lruzicki

Thanks, I've already changed that

**Review by olewandowski1 — APPROVED** [review]
LGTM.

If possible, ask for a review of Damian and/or Dragos as well.

**Review by delcroip — APPROVED** [review]

**Pull request merged**